changelogs/assets-global-css.yml

```
@@ -0,0 +1,22 @@
+Added:
+  - project: Assets
+    component: NPM package
+    description: Added functionality to generate css files based on scss files placed under packages/assets/build. A minified version of each scss file is also provided as [scss-filename].min.css. (#1240)
+    issue: 
+    impact: Minor
+Changed:
+  - project: Assets
+    component: NPM package
+    description: The functionality of packages/assets/scss/00-base/_global.scss has changed to contain optional miscellaneous styles a site can include once for setting up layout/fonts/normalization. (#1240)
+    issue: 
+    impact: Major
```

```suggestion
    impact: Minor
```
major will create a major version in release